mdk/build.gradle

```diff
@@ -19,6 +19,8 @@ archivesBaseName = 'modid'
 
 sourceCompatibility = targetCompatibility = compileJava.sourceCompatibility = compileJava.targetCompatibility = '1.8' // Need this here so eclipse task generates correctly.
 
+sourceSets.main.resources { srcDir 'src/generated/resources' }
```

Comment:
This requires both `srcDir 'src/generated/resources'` as well as `srcDir 'src/main/resources'`.
Since right now you only allow the generated resources to be used.

Reply:
This line is adding the generated resources to  sourceset, not replacing. The normal resources are in the sourceset by default, adding them a second time would be redundant.

Reply:
See that is what i am not sure about. I remember needing += as an operator inbetween.
Let me check.

Reply:
Checked the API, this is indeed correct: https://github.com/gradle/gradle/blob/master/subprojects/core-api/src/main/java/org/gradle/api/file/SourceDirectorySet.java#L50